cmake/detect-intrinsics.cmake

```diff
@@ -25,7 +111,7 @@ macro(check_avx2_intrinsics)
             set(AVX2FLAG "-mavx2")
         endif()
     endif()
-    # Check whether compiler supports AVX2 intrinics
+    # Check whether compiler supports AVX2 intrinsics
```

Comment:
This should be separated as own pull request, possibly grouped with other AVX2 related fixes.

Reply:
Are you referring to this file or a different one?  The check for AVX512 intrinsics is necessary for using this code, so I think you must have meant a different file.

Reply:
Just the spelling fix.

Reply:
Ahh, you want me to back out the spelling fix, then?  I can, it just seemed like the thing to do while I was there and given that it didn't have any affect on the compiled binary, it was probably harmless.

Reply:
Spelling fixes don't need 2 long-time contributors to approve, so they can get merged in faster than code additions or modifications.

Reply:
gotcha